### ISSUE TITLE
quadGate.tmix gating method 

### ISSUE THREAD

**Issue by anaraserikbek** [RGLab/openCyto#116]
Hi everyone,
**I have the following 2 questions about the quadGate.tmix gating method:** 
1)  I've been trying to use `quadGate.tmix` gating method. However, when I use the gatingset object's already gated population & use autoplot on it to further combine with quadGate.tmix results, I receive an Error: 

```
Error: Objects of type flowFrame not supported by autoplot.
> traceback()
3: stop("Objects of type ", paste(class(object), collapse = "/"), 
       " not supported by autoplot.", call. = FALSE)
2: autoplot.default(fr, chnl[1], chnl[2])
1: autoplot(fr, chnl[1], chnl[2]
```

this is the code I'm using: 

```
> first_flow_frame=getData(gs[[1]],"LT CD4+")
> chnl=c("CD25","CD127")
> plotting=autoplot(first_flow_frame,chnl[1],chnl[2])

```

2) Also, when I try to combine the `quadGate.tmix` filter result by geom_gate + autoplot as in the vignette, i get an Error: 

```
Error: could not find function "geom_gate"
> ?geom_gate 
```

& when I try to look for ?geom_gate or ??geom_gate or traceback() my issue, there's nothing. Where does the geom_gate come from? (P.S. I've found that it comes from ggcyto package, however I've tried to install the package & it says it's not available for my version of R 3.2.5. Are there any other functions with which I can plot the filter result of quadGate.tmix?)

3) Also, when specifying the quadGate.tmix in the gating .csv template, how can you specify the plotting method for it? do you write in the gating_args column the autoplot(x) +geom_gate(x)?

**Comment by gfinak** [comment]
1)  Don't call getDataand, use the gatingset directly and see the following on how to use ggcyto

https://www.dropbox.com/s/pa4x4uj2j6jknmc/Finak_ggcyto_June_15.pdf?dl=1

2) clearly ggcyto is either not loaded or not installed. Are you installing it via biocLite or from our github repository? If one doesn't work for you, try the other. Or just download the source package and install it.

3) you don't specify plotting methods for anything in a  CSV template of OpenCyto.

**Comment by mikejiang** [comment]
`ggcyto` was released under `BioC 3.3`, which requires `R >=3.3`. That is why you can install it through normal way (`biocLite`).
So ideally, you want to upgrade your R to `3.3` in order to make sure all the packages are up to date.
Otherwise, you can install it by
```r
devtools::install_github("RGLab/ggcyto", ref = "trunk")
```
Note that it does not guarantee all the dependency packages are in their proper versions as `biocLite` does, which may or may not affect your usage of `ggcyto`.

As @gfinak said,  if gates are already in `GatingSet`, you don't need to extract data using `getData` and plot the `flowSet/flowFrame`. Instead plot `GatingSet`  directly, e.g.
```r
autoplot(gs, "YOUR GATE NAME")
```
